config/cscs.py

```diff
@@ -201,6 +201,13 @@ class ReframeSettings:
                     'cxx': 'mpicxx',
                     'ftn': 'mpif90',
                 },
+                'PrgEnv-gdr': {
+                    'type': 'ProgEnvironment',
+                    'modules': ['gmvapich2/17.02_cuda_8.0_gdr'],
```

Comment:
This is not a full toolchain.
Since our toolchains can be used on any test, they should include the following: ` FFTW, Compiler, MPI, BLAS, BLACS and ScaLAPACK`,
while `gmvapich2` only includes `Compiler, MPI`.

Reply:
That's a good hint for the problem in #85. It might be coming somewhere from the modules backend.

Reply:
So should I change the `type` key?

Reply:
@victorusu what should the additional modules to be added here?

Reply:
@victorusu Since the problem is not on the ReFrame's side or on the test's, I suggest accepting this PR as-is. As soon as the prog. env inconsistency is fixed on the system side, we just need to update the settings files. Unless you see sth here that I'm missing.